src/plonk/mod.rs

```diff
@@ -81,22 +83,10 @@ pub struct PlonkWitness<F: PrimeField> {
 #[derive(Clone, Debug, PartialEq, Eq)]
 pub struct RelaxedPlonkInstance<C: CurveAffine> {
     pub(crate) W_commitment: C,
-    pub(crate) instance: Vec<C::ScalarExt>,
     pub(crate) E_commitment: C,
-    pub(crate) u: C::ScalarExt,
-    pub(crate) y: C::ScalarExt,
-}
-
-impl<C: CurveAffine> Default for RelaxedPlonkInstance<C> {
-    fn default() -> Self {
-        Self {
-            W_commitment: C::from_xy(C::Base::ZERO, C::Base::ZERO).unwrap(),
-            instance: vec![C::ScalarExt::ZERO, C::ScalarExt::ZERO], // TODO Fix Me
-            E_commitment: C::from_xy(C::Base::ZERO, C::Base::ZERO).unwrap(),
-            u: C::ScalarExt::ONE,
-            y: C::ScalarExt::ZERO,
-        }
-    }
+    pub(crate) instance: Vec<C::ScalarExt>,
```

Comment:
Do we have the expected value of instance now? If there is, it should be displayed at least in rustdoc